src/bilder/components/hashicorp/templates/consul-template.service.j2

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=consul agent
 Requires=network-online.target
-After=network-online.target
+After=network-online.target vault.service consul.service
```

Comment:
Is the `vault.service` for the vault agent running on the consul cluster instances?

Reply:
Correct